Build of janus-gateway/* is failing

API seems been changed
```
github.com/pion/example-webrtc-applications/janus-gateway/streaming
# github.com/pion/example-webrtc-applications/janus-gateway/streaming
janus-gateway/streaming/main.go:39:41: handle.Id undefined (type *janus.Handle has no field or method Id, but does have ID)
janus-gateway/streaming/main.go:43:38: handle.Id undefined (type *janus.Handle has no field or method Id, but does have ID)
janus-gateway/streaming/main.go:45:41: handle.Id undefined (type *janus.Handle has no field or method Id, but does have ID)
github.com/pion/example-webrtc-applications/janus-gateway/video-room
# github.com/pion/example-webrtc-applications/janus-gateway/video-room
janus-gateway/video-room/main.go:21:43: handle.Id undefined (type *janus.Handle has no field or method Id, but does have ID)
janus-gateway/video-room/main.go:25:40: handle.Id undefined (type *janus.Handle has no field or method Id, but does have ID)
janus-gateway/video-room/main.go:27:43: handle.Id undefined (type *janus.Handle has no field or method Id, but does have ID)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,7 +17,7 @@ cache:
 
 before_install:
   - sudo apt-get install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
-  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v1.18.0
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v1.19.1
   - sh -c "cd c-data-channels && make"
 
 install:

--- a/internal/gstreamer-sink/gst.go
+++ b/internal/gstreamer-sink/gst.go
@@ -1,3 +1,4 @@
+// Package gst provides an easy API to create an appsrc pipeline
 package gst
 
 /*

--- a/internal/gstreamer-src/gst.go
+++ b/internal/gstreamer-src/gst.go
@@ -1,3 +1,4 @@
+// Package gst provides an easy API to create an appsink pipeline
 package gst
 
 /*

--- a/janus-gateway/streaming/main.go
+++ b/janus-gateway/streaming/main.go
@@ -36,17 +36,16 @@ func watchHandle(handle *janus.Handle) {
 		msg := <-handle.Events
 		switch msg := msg.(type) {
 		case *janus.SlowLinkMsg:
-			fmt.Print("SlowLinkMsg type ", handle.Id)
+			fmt.Print("SlowLinkMsg type ", handle.ID)
 		case *janus.MediaMsg:
 			fmt.Print("MediaEvent type", msg.Type, " receiving ", msg.Receiving)
 		case *janus.WebRTCUpMsg:
-			fmt.Print("WebRTCUp type ", handle.Id)
+			fmt.Print("WebRTCUp type ", handle.ID)
 		case *janus.HangupMsg:
-			fmt.Print("HangupEvent type ", handle.Id)
+			fmt.Print("HangupEvent type ", handle.ID)
 		case *janus.EventMsg:
 			fmt.Printf("EventMsg %+v", msg.Plugindata.Data)
 		}
-
 	}
 }
 

--- a/janus-gateway/video-room/main.go
+++ b/janus-gateway/video-room/main.go
@@ -18,19 +18,17 @@ func watchHandle(handle *janus.Handle) {
 		msg := <-handle.Events
 		switch msg := msg.(type) {
 		case *janus.SlowLinkMsg:
-			log.Println("SlowLinkMsg type ", handle.Id)
+			log.Println("SlowLinkMsg type ", handle.ID)
 		case *janus.MediaMsg:
 			log.Println("MediaEvent type", msg.Type, " receiving ", msg.Receiving)
 		case *janus.WebRTCUpMsg:
-			log.Println("WebRTCUp type ", handle.Id)
+			log.Println("WebRTCUp type ", handle.ID)
 		case *janus.HangupMsg:
-			log.Println("HangupEvent type ", handle.Id)
+			log.Println("HangupEvent type ", handle.ID)
 		case *janus.EventMsg:
 			log.Printf("EventMsg %+v", msg.Plugindata.Data)
 		}
-
 	}
-
 }
 
 func main() {
@@ -152,5 +150,4 @@ func main() {
 	}
 
 	select {}
-
 }

--- a/sfu-ws/main.go
+++ b/sfu-ws/main.go
@@ -18,7 +18,6 @@ func checkError(err error) {
 }
 
 func init() {
-
 	// Generate pem file for https
 	genPem()
 
@@ -31,7 +30,6 @@ func init() {
 
 	// Create the API object with the MediaEngine
 	api = webrtc.NewAPI(webrtc.WithMediaEngine(m))
-
 }
 
 func main() {

--- a/sfu-ws/room.go
+++ b/sfu-ws/room.go
@@ -52,7 +52,6 @@ const (
 )
 
 func room(w http.ResponseWriter, r *http.Request) {
-
 	// Websocket client
 	c, err := upgrader.Upgrade(w, r, nil)
 	checkError(err)
@@ -79,7 +78,6 @@ func room(w http.ResponseWriter, r *http.Request) {
 
 		pubReceiver.OnTrack(func(remoteTrack *webrtc.Track, receiver *webrtc.RTPReceiver) {
 			if remoteTrack.PayloadType() == webrtc.DefaultPayloadTypeVP8 || remoteTrack.PayloadType() == webrtc.DefaultPayloadTypeVP9 || remoteTrack.PayloadType() == webrtc.DefaultPayloadTypeH264 {
-
 				// Create a local video track, all our SFU clients will be fed via this track
 				var err error
 				videoTrackLock.Lock()
@@ -107,9 +105,7 @@ func room(w http.ResponseWriter, r *http.Request) {
 						checkError(err)
 					}
 				}
-
 			} else {
-
 				// Create a local audio track, all our SFU clients will be fed via this track
 				var err error
 				audioTrackLock.Lock()
@@ -156,7 +152,6 @@ func room(w http.ResponseWriter, r *http.Request) {
 			})
 		})
 	} else {
-
 		// Create a new PeerConnection
 		subSender, err := api.NewPeerConnection(peerConnectionConfig)
 		checkError(err)

--- a/sfu-ws/web.go
+++ b/sfu-ws/web.go
@@ -14,7 +14,6 @@ import (
 )
 
 func genPem() {
-
 	privateKey, err := rsa.GenerateKey(rand.Reader, 2048)
 	checkError(err)
 

